src/Core/AdminConsole/Models/Data/EventIntegrations/WebhookIntegrationConfiguration.cs

```diff
@@ -2,4 +2,4 @@
 
 namespace Bit.Core.AdminConsole.Models.Data.EventIntegrations;
 
-public record WebhookIntegrationConfiguration(string url);
+public record WebhookIntegrationConfiguration(string Url, string? Scheme = null, string? Token = null);
```

Comment:
🎨 After you changed the parameter order, it made me think about any accidental swaps of your intended values in the commit. I don't see any of that, but if this first parameter were a `Uri` it would be a compile error at least. I am not sure of the impact of that addition and it would be extra work on the parsing side (and perhaps useless).